### ISSUE TITLE
[WARNING] Nan problem in PostNet of FastSpeech/FastSpeech2 when using Multi-GPU.

### ISSUE THREAD

**Issue by dathudeptrai** [TensorSpeech/TensorFlowTTS#187]
I just found that `tf.keras.layers.experimental.SyncBatchNormalization` in multi-gpu can yield `nan` loss sometimes. There are some issue already in github (https://github.com/tensorflow/tensorflow/issues/41980). The workaround right now is simply remove postnet in FastSpeech/FastSpeech2 (if you get nan before the model convergence :D, in my case, i got nan after model convergence so everything fine :D), it doesn't hurt the performance :D. We will make the base_trainer handle the nan loss later :D.

**Comment by jucaowei** [comment]
sigle_gpu got the same problem

**Comment by dathudeptrai** [comment]
> sigle_gpu got the same problem

Disble mixed precision will
Fix ur problem.

**Comment by jucaowei** [comment]
thanks
